### ISSUE TITLE
End-to-bridge encryption

### ISSUE THREAD

**Issue by tulir** [hifi/heisenbridge#113]

**Comment by Ulrar** [comment]
Hi,

I was very confused setting this up for a while there, and I just finally figured out that my issue was encryption.

I may have missed it if it's already there, but it might be worth mentioning in the README that encryption has to be disabled for this to work, so `encryption_enabled_by_default_for_room_type: all` has to be removed from synapse's config file or the bridge won't answer private messages, and no errors are displayed on either side.

**Comment by hifi** [comment]
Yeah it really should be in the README, I've completely forgotten about that.

Is the option on by default now?

**Comment by Ulrar** [comment]
Not sure if it's by default, sounds like something I probably enabled myself to be honest, but it's been there for so long I forgot about it.
Took me a while to realize that could be why I wasn't getting any answers from the bridge. A note would be great, thanks!

Oct 16, 2021 15:05:07 Toni Spets ***@***.***>:

**Comment by regulad** [comment]
Bump. E2EE/E2BE would be a fantastic addition to heisenbridge. Currently, the `mautrix` bridges support E2BE, and with key deletion it would add much-needed privacy.
